### ISSUE TITLE
Progression Screen background glitch

### ISSUE THREAD

**Issue by MarianaBoni** [deathlyrage/theisle-bugs#351]
Patch: 4856
Server: Test Server/Hosted
When selecting a dinosaur on the Progression Menu, that dinosaur will appear in the background, behind the dinosaur options.
When logging out and in again, the same dinosaur will appear in the background as default, only changing when clicking on another one.
https://youtu.be/keMeF0XkFnk

**Comment by dinosauriac** [comment]
I was curious about this since I've never played progression. There's the same functionality there as in sandbox / survival for showing the dino preview and selecting skins. However, the way I saw it was setup initially, none of that was visible at all. So I tried out having it be in the background there to see what it'd be like. 

If the issue you're talking about is more that the dino selection doesn't clear when you log back in, then there's probably a fix for that I can find. I guess it might be kind of a moot point if the game mode will likely be removed sooner rather than later. Not sure how to proceed.
